normalizer.py

```diff
@@ -38,16 +42,37 @@ def main(argv):
               normalizer_lib.token_normalizer(FLAGS.string_to_normalize))
         print("SENTENCE_BASED:\t"+
               normalizer_lib.sentence_normalizer(FLAGS.string_to_normalize))
-
     else:
+        data_source: str = FLAGS.data_source
+        if data_source == "ud":
+            infile = language.UD
+        elif data_source == "um":
+            infile = language.UM
+        elif data_source == "ac":
+            infile = language.AC
+        elif data_source == "oscar":
+            infile = language.OSCAR
+        elif data_source == "lcc":
+            infile = language.LCC
+        try:
+            input_text: List[str] = preprocess.process_data(infile, FLAGS.data_source)
+        except Exception:
+            print("No data file from '{}' for '{}'".format(data_source, FLAGS.language))
```

Comment:
Consider using f-strings.

Reply:
Good idea! I hadn't heard of f-strings before, but they are definitely cleaner than print .format.